benchmarks/nel/project.yml

```diff
@@ -9,8 +9,10 @@ vars:
   gpu_id: ""
   use_filtered_dumps: "--use_filtered_dumps"
   training_max_steps: 1000
+  run: ""
+  eval_comp_highlight_metric: "F-score"  # one of ("F-score", "Recall", "Precision")
```

Comment:
Maybe just `eval_highlight_metric`

Reply:
Also, perhaps the options should just be "F", "p" and "r" to prevent typos? (note that one of the docstrings has "f1" instead of "F-score")

Reply:
Renamed. Where did you see "f1"? Wasn't able to find it :thinking:

Reply:
It's removed now :-) https://github.com/explosion/projects/pull/129/commits/0df2ca989a0409139e37bf1ab07bb8096b6518e3#diff-354e564d383eafab258d09191b03c1eb1d98735fafe655a01289db50c120d80dL261